lib/private/Http/Client/Client.php

```diff
@@ -81,7 +83,8 @@ private function buildRequestOptions(array $options): array {
 		$options = array_merge($defaults, $options);
 
 		if (!isset($options[RequestOptions::HEADERS]['User-Agent'])) {
-			$options[RequestOptions::HEADERS]['User-Agent'] = 'Nextcloud Server Crawler';
+			$userAgent = 'Nextcloud Server Crawler ' . $this->serverVersion->getVersionString();
```

Comment:
While on it let's align with global standards?
```suggestion
			$userAgent = 'Nextcloud-Server-Crawler/' . $this->serverVersion->getVersionString();
```